README.md

```diff
@@ -67,9 +53,7 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+I want to thank Cindy Shin for providing the design. You can have more information about him [here](https://www.behance.net/adagio07).
```

Comment:
Kudos for giving appropriate credit to the author of the original design :+1:

Reply:
Can you please give more details, about what you mean by this statement?

Reply:
I congratulate you for doing the correct thing :smile:

Reply:
Yeah, thank you. 
I did not know that, I thought there is an issue with that. :laughing:  :rofl: